core/src/components/picker-column/test/standalone/picker-column.e2e.ts

```diff
@@ -18,3 +18,27 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
     });
   });
 });
+
+/**
+ * ion-picker-column does not have mode-specific styling for document direction
+ */
+configs({ modes: ['ios'] }).forEach(({ title, config, screenshot }) => {
+  test.describe(title('picker-column: document direction'), () => {
```

Comment:
It looks like we have RTL screenshots in `picker/test/basic` (the screenshots that got updated). What does this test do differently?

Reply:
Not really, I was on the fence of whether it would be beneficial. I figured that it would be rejected during a review if that was the case. I'll remove it.